lib/daterangepicker.js

```diff
@@ -100,7 +100,7 @@
             options.template = '<div class="daterangepicker dropdown-menu">' +
                 '<div class="calendar left">' +
                     '<div class="daterangepicker_input">' +
-                      '<input class="input-mini" type="text" name="daterangepicker_start" value="" />' +
+                      '<input class="input-mini" type="text" name="daterangepicker_start" value="" disabled/>' +
```

Comment:
Can make these disabled based on an option but thought that was misguided since it would cause a bug any time somebody forgot to include it and used a DateRangePicker in a non en-US locale